config/webpack.server.js

```diff
@@ -6,19 +6,48 @@ const merge = require('webpack-merge');
 const defaults = require('./webpack.defaults.js');
 const packageConfig = require('./packageConfig');
 
-module.exports = merge.smart({
-  module: {
-    rules: [{
-      test: /\.css$/,
-      use: [path.join(__dirname, '../lib/exportLocalsLoader.js')],
-    }, {
-      test: /\.scss$/,
-      use: [path.join(__dirname, '../lib/exportLocalsLoader.js')],
-    }],
-  },
-}, defaults, {
+const config = merge.smart(defaults, {
   target: 'node',
   mode: 'production',
+  module: {
+    rules: [
+      {
+        test: /\.css$/,
+        include: path.resolve('src'),
+        use: [
+          {
+            loader: 'css-loader',
+            options: {
+              exportOnlyLocals: true,
+            },
+          },
+        ],
+      },
+      {
+        test: /\.css$/,
+        include: path.resolve('node_modules'),
```

Comment:
this and the previous test appear to be the same, can include take an array of paths?